### PR TITLE
tableview pagination

### PR DESCRIPTION
**Pull request by rahulbhate**
1) Changes made for better pagination in table-view. Shows page in the following format : 
{First item number of the current page}-{last item number of the current page} of {total number of items}.
2) Removed hardcoded "Total number of records" from the table-view component.

Tested with versions : 
Alyson-v3 : release-2.0.1
Qwanda : 2.0.2
With project PCSS.

### CODE REVIEW

File: src/views/components/table/Table.web.js [hunk 1]
```diff
@@ -489,22 +458,25 @@ class TableView extends Component {
...
                 >
                   <Box
-                    backgroundColor={currentPage <= 0 ? '#ddd' : '#5173c6'}
+                    backgroundColor={currentPage <= 0 ? '#ddd' : '#d16360'}
                     padding={10}
                   >
                     <Text
-                      text="Previous"
-                      color="white"
+                      text="<"
```
Comment (conorpirie) [inline src/views/components/table/Table.web.js]: Use an Icon instead of text here.

File: src/views/components/table/Table.web.js [hunk 2]
```diff
@@ -513,12 +485,13 @@ class TableView extends Component {
...
                 >
                   <Box
-                    backgroundColor={currentPage + 1 >= this.state.totalPages ?  '#ddd' : '#5173c6'}
+                    backgroundColor={currentPage + 1 >= this.state.totalPages ?  '#ddd' : '#d16360'}
                     padding={10}
                   >
                     <Text
-                      text={isLoadingNextPage || isSearching ? 'Loading...' : 'Next'}
-                      color="white"
+                      text={isLoadingNextPage || isSearching ? 'Loading...' : '>'}
```
Comment (conorpirie) [inline src/views/components/table/Table.web.js]: Use the ternary operator to choose between a Text component with text="Loading..." and an Icon for the arrow

File: src/views/components/table/Table.web.js (hunk 1 above)
Comment (rahulbhate) [reply src/views/components/table/Table.web.js]: @conorpirie Calvin says "No need icons for previous or next buttons, just << or >> text is sufficient"

File: src/views/components/table/Table.web.js (hunk 2 above)
Comment (ngprnk) [reply src/views/components/table/Table.web.js]: @conorpirie  should i close this pull request ?

**Pull request closed without merging**